CHANGELOG.md

```diff
@@ -0,0 +1,18 @@
+#CHANGE LOG
```

Comment:
This `CHANGELOG` should be empty (only with the default template), can you explain me why ?

Reply:
Because the project has not yet been released.